src/Search/Search.js

```diff
@@ -14,43 +17,109 @@ function WikipediaSearch() {
     if (searchInput.trim() !== '') {
       displaySearchResults(searchInput);
     }
+    setSearchInput('');
   };
 
-  const displaySearchResults = (searchTerm) => {
-    let url = `https://en.wikipedia.org/w/api.php?action=query&list=search&prop=info&inprop=url&utf8=&format=json&origin=*&srlimit=20&srsearch=${searchTerm}`;
-  
-    fetch(url)
+  async function displaySearchResults(searchTerm){
+    // let url = `https://en.wikipedia.org/w/api.php?action=query&list=search&prop=info&inprop=url&utf8=&format=json&origin=*&srlimit=20&srsearch=${searchTerm}`;
+    const url = 'https://en.wikipedia.org/w/api.php?'
+    let params = {
+      action: 'query',
+      list: 'search',
+      format: 'json',
+      origin: '*',
+      srlimit: 20,
+      srsearch: searchTerm,
+    };
+
+    await fetch(url + new URLSearchParams(params).toString())
       .then((response) => {
         if (!response.ok) {
           throw new Error('Network response was not ok');
         }
         return response.json();
       })
       .then((data) => {
-        const filteredResults = data.query.search.filter(
-          (item) =>
-            item.title.toLowerCase().includes('abuse') ||
-            item.snippet.toLowerCase().includes('abuse')||
-            item.snippet.toLowerCase().includes('allegation') ||
-            item.snippet.toLowerCase().includes('drug') ||
-            !containsHtmlTags(item.snippet.toLowerCase()) // Check if snippet contains HTML tags
-        );
-        // Clean up the snippet text by removing HTML tags and their content
-        filteredResults.forEach((result) => {
-          result.snippet = result.snippet.replace(/<[^>]*>?/gm, '');
-        });
-        setResults(filteredResults);
-        console.log(filteredResults)
+// Sets the state of the initial results to the first result of the search (i.e. Rush Limbaugh)
+        setInitialResults(data.query.search[0])
```

Comment:
good solution to this, loved the use of the first position in the array, it makes for a clean result